release-calendar/src/types.ts

```diff
@@ -26,6 +26,20 @@ export enum Channel {
   CREATED = 'created',
 }
 
+interface Inames {
+  [key: string]: string;
+}
+
+export const channelNames: Inames = {
```

Comment:
`Inames` look a little weird. Is that what typescript recommends?

Reply:
Typescript wants the first letters of words to be capitalized. I'll switch it to `Names` . It seems like people on forums online are very opinionated about it. :)